lib/reporters/html.js

```diff
@@ -58,6 +59,20 @@ var playIcon = '&#x2023;';
  */
 function HTML(runner, options) {
   Base.call(this, runner, options);
+  for (var opt in options) {
+    if (options.hasOwnProperty(opt) && opt === 'reporterOptions') {
+      if (
+        options[opt] &&
+        (options[opt].highlight ||
+          typeof options[opt].highlight === 'undefined')
+      ) {
+        options[opt].highlight = true;
+      }
+      if (!options[opt]) {
+        options[opt] = {highlight: true};
+      }
+    }
+  }
```

Comment:
Can we simplify this? 
```
var enableHighlight = true;
if (
  options.reporterOptions &&
  options.reporterOptions.hasOwnProperty('highlight') &&
  !options.reporterOptions.highlight
) {
  enableHighlight = false;
}
```

Reply:
yeah, that's way better. thank you I really appreciate it!